src/components/ScholarshipDetailCard.jsx

```diff
@@ -97,58 +72,72 @@ export default function ScholarshipDetailCard({ scholarship, preview }) {
     }
   };
 
-  function DetailCardCell({ label, text, bottom, top }) {
+  function DetailCardCell({ label, text }) {
     return (
       <>
-        {top && <Divider light className={classes.divider} />}
         <Grid container justifyContent="space-between">
           <Grid item xs={12} sm>
             <Typography>{label}</Typography>
           </Grid>
-          <Grid item className={classes.cardDetailText} xs={12} sm>
+          <Grid item sx={{ textAlign: { sm: 'right' } }} xs={12} sm>
             <Typography>{text}</Typography>
           </Grid>
         </Grid>
-        {bottom && <Divider light className={classes.divider} />}
+        <Divider light sx={{ m: 1.5 }} />
       </>
     );
   }
 
   DetailCardCell.propTypes = {
     label: PropTypes.string.isRequired,
     text: PropTypes.oneOfType([PropTypes.string, PropTypes.number]).isRequired,
-    bottom: PropTypes.bool,
-    top: PropTypes.bool,
-  };
-  DetailCardCell.defaultProps = {
-    bottom: false,
-    top: false,
   };
 
   return (
-    <Card className={classes.detailSection}>
-      <Typography variant="h4">{name}</Typography>
-      <Typography variant="h5" gutterBottom>
+    <Card sx={{ p: 3 }}>
+      <Typography variant="h6" gutterBottom>
         {organization}
       </Typography>
+      <Typography variant="h4" gutterBottom>
+        {name}
+      </Typography>
       {preview && (
         <MuiLink
           href="#"
           underline="always"
-          onClick={(e) => e.preventDefault()}>
+          onClick={(e) => e.preventDefault()}
+          sx={{ display: 'block', mb: 2 }}>
           {website}
         </MuiLink>
       )}
 
-      <Box className={classes.actionSection}>
+      <Grid container spacing={3}>
+        <Grid item>
+          <Box sx={{ display: 'flex' }}>
+            <AttachMoneyIcon color="primary" />
+            <Typography>{ScholarshipAmount.toString(amount)}</Typography>
```

Comment:
There's a double $ sign:
![image](https://user-images.githubusercontent.com/8023233/145146079-3db0712c-f66a-4070-8490-21deb3df9491.png)

Maybe we should change the `toString()` method?

Reply:
done. removed the dollar sign